build.gradle.kts

```diff
@@ -0,0 +1,16 @@
+
+import com.saveourtool.processbuilder.configureDetekt
+import com.saveourtool.processbuilder.configureDiktat
+import com.saveourtool.processbuilder.createDetektTask
+
+group = "com.saveourtool"
+description = "Multiplatform ProcessBuilder implementation"
```

Comment:
just a not to naming: maybe just `Kotlin Process Builder`?

Reply:
Ok, will do it

Reply:
Done